avm/res/insights/private-link-scope/main.bicep

```diff
@@ -371,32 +350,36 @@ type privateEndpointType = {
 
   @description('Optional. Specify if you want to deploy the Private Endpoint into a different resource group than the main resource.')
   resourceGroupName: string?
-}[]?
+}
 
+@export()
+@description('The scoped resource type.')
 type scopedResourceType = {
   @description('Required. Name of the private link scoped resource.')
   name: string
 
   @description('Required. The resource ID of the scoped Azure monitor resource.')
   linkedResourceId: string
-}[]?
+}
 
+@export()
+@description('The access mode type.')
 type accessModeType = {
-  @description('Optional. List of exclusions that override the default access mode settings for specific private endpoint connections. Exclusions for the current created Private endpoints can only be applied post initial provisioning.')
+  @description('Required. List of exclusions that override the default access mode settings for specific private endpoint connections. Exclusions for the current created Private endpoints can only be applied post initial provisioning.')
   exclusions: {
-    @description('Required. The private endpoint connection name associated to the private endpoint on which we want to apply the specific access mode settings.')
-    privateEndpointConnectionName: string
+    @description('Optional. The private endpoint connection name associated to the private endpoint on which we want to apply the specific access mode settings.')
+    privateEndpointConnectionName: string?
```

Comment:
Why did both change the state?

Reply:
I'm not sure I'm following (and the same goes for the other parameter): The parameter used to be `Required` and was now changed to `Optional`. The static tests must have failed if e.g. only the description or only the type was changed to either required/optional but not both.
So my question is rather why this parameter was fundamentally changed to be optional even though it used to be required, and why the `exclusions` are not required even though they are implemented as being optional in the template above: 
![image](https://github.com/user-attachments/assets/cd6b59a9-ac0a-4692-9ab6-84169a69177f)